### ISSUE TITLE
OSC Slideshow pause command

### ISSUE THREAD

**Issue by Total-Image** [phuvf/oscpoint#27]
/oscpoint/slideshow/pause

**Comment by phuvf** [comment]
Not really sure what you mean by this - is this for use when slides advance automatically by time?

**Comment by Total-Image** [comment]
Yes that is correct. Some presenters use the advance slides by x seconds and want to pause / continue @ a certain cue.
If I'm right then there is a viewstate PPTSlideShowPaused option in the PPT interop. Would be very usefull if you could implement this.
